src/style.css

```diff
@@ -0,0 +1,363 @@
+
+/*STYLES THE OVERAL LAYOUT*/
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
+    position: relative;
+}
+
+#main-container {
+    margin: 0 5%;
+    height: 100vh;
+    display: grid;
+    grid-template-columns: 1fr 2fr 1fr;
+}
+
+section {
+    padding: 10px;
+    display: grid;
+    grid-template-rows: 40px 1fr;
+}
+
+footer {
+    font-size: 10px;
+    height: fit-content;
+}
+
+
+
+/*RECURRENT STYLES*/
+
+.rounded-container {
+    background-color: rgb(39, 175, 209);
+    height: 20px;
+    color: white;
+    width: 92%;
+    border-radius: 40px;
+    text-align: center;
+    padding: 6px 0;
+}
+
+
+.profile-pic-container {
+    display: block;
+    width: 40px;
+    height: 40px;
+    border-radius: 40px;
+    border: 1px dotted grey;
+    background-image: url("../assets/img/pexels-aisling-kerr-16904943.jpg");
+    background-size: contain;
+}
+
+.profile-pic-container > img {
+    width: 50px;
+    height: auto;
+    object-fit: cover;
+}
+
+.icon {
+    height: 23px;
+    place-self: center;
+}
+
+h2 {
+    display: inline-block;
+    font-size: 18px;
+    margin-bottom: 5px;
+    font-family: "Tahoma", "Arial", sans-serif;
+    font-weight: bolder;
+    margin-left: 10px;
+ }
+
+ h3 {
+    display: inline-block;
+    font-size: 12px;
+    padding-bottom: 5px;
+    font-family: "Tahoma", "Arial", sans-serif;
+    font-weight: bolder;
+    width:fit-content;
+ }
+
+a {
+    color: rgb(24, 107, 128);
+    font-size: 10px;
+}
+
+p {
+    font-size: 10px;
+}
+
+::-webkit-scrollbar {
+    border: none;
+}
+
+/*STYLES THE LEFT-MENU*/
+
+#clone-icon {
+    height: 40px;
+}
+
+#left-menu {
+    padding-left: 50px;
+    height: calc(100vh - 20px);
+}
+
+#left-menu main {
+    margin-top: 10px;
+    display: grid;
+    grid-template-rows: repeat(8, 40px) 50px;
+    gap: 10px;
+    height:fit-content;
+}
+
+#left-menu main > div {
+    padding: 5px; 
+    display: grid;
+    grid-template-columns: 25px 1fr;
+    
+}
+
+#left-menu main .rounded-container{
+    display: inline-block;
+    max-width: 250px;
+    height: 40px;
+    place-self: center;
+}
+
+#left-menu main .rounded-container h2 {
+    display: inline-block;
+    width: fit-content;
+    place-self: center;
+    margin: 10px 0 0 0;
+}
+
+
+#left-menu footer {
+    display: grid;
+    grid-template-columns: 40px 1fr 40px;
+    height: 40px;
+    position: sticky;
+    bottom: 5px;
+    padding: 5px;
+}
+#left-menu footer > div:nth-child(2) {
+    padding: 3px 0 0 5px;
+}
+
+#left-menu footer > div:nth-child(3) {
+    padding: 10px 5px;
+}
+
+/*hover and selected styles for LEFT-MENU*/
+#left-menu main > div:hover {
+    color: orange;
+}
+#left-menu main > div:active {
+    background-color: rgba(255, 166, 0, 0.6);
+    color: white;
+}
```

Comment:
I like the hover stuff!

The active selector is quite quick though and quite easily missed. I think it would be worth to take a look at whether you can use [transitions](https://developer.mozilla.org/en-US/docs/Web/CSS/transition) to make them linger longer.

Reply:
thanks for that :) nice to get some new stuff to use, I'll definitely look into it!